EIPS/eip-xxxx.md

```diff
@@ -0,0 +1,153 @@
+---
+eip: <TBD>
+title: Genesis File Format Standardization
+author: Justin Florentine (@jflo) <justin@florentine.us>
+discussions-to: https://ethereum-magicians.org/t/eip-xxxx-genesis-json-standardization/24271
+status: Draft
+type: Informational
+category: Core
+created: 2025-05-19
+requires: 0
+---
+
+## Simple Summary
+
+Standardize the format of the `genesis.json` file used to initialize modern (post-merge) Ethereum chains, aligning it with Geth’s current implementation and providing a JSON Schema for validation.
+
+## Abstract
+
+This EIP defines a canonical structure for Ethereum genesis files (`genesis.json`) used to bootstrap Ethereum networks. The standard aligns with the de facto structure implemented by Geth (Go-Ethereum) and introduces a JSON Schema to ensure consistency and tool compatibility across clients.
+
+## Motivation
+
+The lack of an official standard for the `genesis.json` file has led to incompatibilities, bugs and confusion, as well as added workload for those running multiple clients together in test networks. This EIP aims to reduce ambiguity by defining a consistent structure and enabling tooling through schema-based validation.
+
+## Specification
+
+The canonical genesis file MUST be a JSON object with the following top-level fields:
+
+### Top-Level Fields
+
+| Field           | Type              | Required | Description                                                     |
+|-----------------|-------------------|----------|-----------------------------------------------------------------|
+| `config`        | `object`          | Yes      | Chain configuration parameters.                                 |
+| `alloc`         | `object`          | No       | Map of addresses to pre-allocated balances and/or code/storage. |
```

Comment:
Geth marks this as required https://github.com/ethereum/go-ethereum/blob/63740b7aca732f6c204c56a9e4b4932f5251e4e0/core/genesis.go#L66